docs/EN_US/ECLLanguageReference/ECLR_mods/SpecStruc-EMBED.xml

```diff
@@ -104,21 +124,28 @@ ENDEMBED</programlisting></para>
   corruption and/or security issues, so great care and forethought are
   advised--consult with Technical Support before using.</emphasis></para>
 
-  <para>Example:</para>
+  <para>Examples:</para>
 
   <programlisting lang="ECL">//First form: a structure
 IMPORT Python3 AS Python;  //make Python language available
 
-INTEGER addone(INTEGER p) := EMBED(Python)
+INTEGER addone(INTEGER p) := EMBED(Python :TIME('MyTime'))
 # Python code that returns one more than the value passed to it
 if p &lt; 10:
   return p+1
 else:
   return 0
 ENDEMBED;
+addone(3);
+addone(11);
+</programlisting>
+
+  <para></para>
 
-//Second form: a function
-INTEGER addtwo(INTEGER p) := EMBED(Python, 'p+2');  </programlisting>
+  <programlisting>//Second form: a function
```

Comment:
Should these be

`<programlisting lang="ECL" role="runnable">
`
?

Reply:
But it should have lang="ECL"?

Reply:
It is a combination of ECL and Python. so we usually leave it off. 
When omitted, it is treated as ECL anyway. IOW, it makes no difference today, but in the future, it could. We only support ECL syntax colorization now.